Framework/DataHandling/src/LoadILLLagrange.cpp

```diff
@@ -138,15 +138,15 @@ void LoadILLLagrange::loadData() {
   nDims = scanVarSpace.getSimpleExtentNdims();
   dimsSize = std::vector<hsize_t>(nDims);
   scanVarSpace.getSimpleExtentDims(dimsSize.data(), nullptr);
-  if ((nDims != 2) || (dimsSize[1] != m_nScans))
+  if ((nDims != 2) || (dimsSize[1] != m_nScans)) // cppcheck-suppress containerOutOfBounds
```

Comment:
@perenon These suppressions are something that you will want to look into once this gets merged.